### PR TITLE
SSO/RBAC Docs update

### LINKED ISSUE #5928: Update RBAC/SSO docs
Update lakeFS docs for enterprise

### PR DESCRIPTION
**Pull request by Isan-Rivkin**
Closes #5928

Updating documentation for lakeFS chart, pending [PR merge](https://github.com/treeverse/charts/pull/177)

### CODE REVIEW

File: docs/enterprise/sso.md
Comment (nopcoder) [inline docs/enterprise/sso.md:79]: Can we move this one as a note outside the config example?
I assume the yaml will be used to copy/paste and this comment is informative for the helm users.

**Review by nopcoder — APPROVED** [review]
LGTM - minor comment related to helm

**Review by Isan-Rivkin — COMMENTED** [review]
Thanks @nopcoder , will merge after the chart is merged.

**Pull request merged**
